src/retrieval/vote_and_verify.cc

```diff
@@ -17,6 +17,7 @@
 #include "retrieval/vote_and_verify.h"
 
 #include <array>
+#include <forward_list>
```

Comment:
Unused include?

Reply:
Aha, yes, left over from the other EIC implementation.